### PR TITLE
VEN[1206] Improve repo quality

### PR DESCRIPTION
**Pull request by UrAvgDeveloper**
## Description

This PR Improves the quality of oracle repo

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: hardhat.config.ts [hunk 1]
```diff
@@ -1,106 +1,31 @@
...
 
 const config: HardhatUserConfig = {
-  defaultNetwork: "hardhat",
-  namedAccounts: {
-    deployer: {
-      default: 0, // here this will by default take the first account as deployer
-    },
-  },
-  etherscan: {
-    apiKey: {
```
Comment (chechu) [inline hardhat.config.ts:67]: Don't we need this section to verify the code in bscscan?

Comment (UrAvgDeveloper) [reply hardhat.config.ts:67]: We need this, added back [here](https://github.com/VenusProtocol/oracle/pull/54/commits/188a7f2040e75727da892d435c0eb1f743f904de)

**Review by 0xlucian — APPROVED** [review]

File: hardhat.config.ts [hunk 2]
```diff
@@ -111,11 +52,83 @@ const config: HardhatUserConfig = {
...
+      {
+        artifacts: "node_modules/@venusprotocol/oracle/artifacts",
+      },
+      {
+        artifacts: "node_modules/@venusprotocol/venus-protocol/artifacts",
+      },
+    ],
+    deployments: {
+      bsctestnet: ["node_modules/@venusprotocol/oracle/deployments/bsctestnet"],
+    },
```
Comment (chechu) [inline hardhat.config.ts:109]: I think we don't need this config

File: README.md [hunk 3]
```diff
@@ -2,117 +2,83 @@
...
-### TypeChain
-
-Compile the smart contracts and generate TypeChain artifacts:
+yarn install
 
-```sh
-$ yarn typechain
 ```
 
-### Linting and code formatting
```
Comment (narayanprusty) [inline README.md:32]: I think we should keep this section on the commands required to detect and fix lint issues. We use these command to fix the lint issues.

Comment (UrAvgDeveloper) [reply README.md:32]: Right but this part is already covered in hardhat commands [here](https://github.com/VenusProtocol/oracle/pull/54/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R44), should we need yarn commands too?

File: hardhat.config.ts (hunk 2 above)
Comment (UrAvgDeveloper) [reply hardhat.config.ts]: Fixed !
Right, no need of this dependency in oracle, if requires in future we will add them back.

**Review by defcon022 — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (chechu) [reply README.md:32]: Do hardhat commands run the linter?

Comment (UrAvgDeveloper) [reply README.md:32]: Yes they do !

Comment (narayanprusty) [reply README.md:32]: The `yarn` commands looks simple and are easier to use. I would prefer to keep them if you guys agree. 

Or else we can create a specific section on how to fix the ts and sol lint issues when pipeline fails due to lint checks. And we can keep the linting specific npx command ordered there.

**Review by chechu — APPROVED** [review]

**Pull request merged**